contracts/modules/PoolController.sol

```diff
@@ -125,14 +128,27 @@ contract PoolController is
         riskpool.decreaseBalance(processId, amount);
     }
 
+    function isArchivingAllowed(uint256 id) external view returns (bool) {
+        IRiskpool riskpool = _getRiskpoolForId(id);
+        uint256 riskpoolId = riskpool.getId();
+        require(
+            _component.getComponentState(riskpoolId) == IComponent.ComponentState.Paused
+            || _component.getComponentState(riskpoolId) == IComponent.ComponentState.Suspended, 
+            "ERROR:POL-010:ARCHIVED_IS_FINAL_STATE"
+            );
+        require(
+            _bundle.unburntBundles(riskpoolId) == 0, 
+            "ERROR:POL-011:RISKPOOL_HAS_BUNDLES"
```

Comment:
suggestion to rename error message to "ERROR:POL-011:RISKPOOL_HAS_UNBURNT_BUNDLES" to provide reason for revert (all normal riskpools would have bundles ...)